src/main/java/uk/gov/hmcts/reform/prl/services/EditReturnedOrderService.java

```diff
@@ -0,0 +1,105 @@
+        log.info("** UserDetails {}", userDetails);
+        Map<String, Object> caseDataMap = new HashMap<>();
+        List<Element<DraftOrder>> supportedDraftOrderList = new ArrayList<>();
+        caseData.getDraftOrderCollection().forEach(
+            draftOrderElement -> {
+                log.info("** order created by email {}", draftOrderElement.getValue().getOtherDetails().getOrderCreatedByEmailId());
+                if (OrderStatusEnum.rejectedByJudge.getDisplayedValue()
+                    .equalsIgnoreCase(draftOrderElement.getValue().getOtherDetails().getStatus())
+
+                    && userDetails.getEmail().equalsIgnoreCase(draftOrderElement.getValue().getOtherDetails().getOrderCreatedByEmailId())) {
+                    supportedDraftOrderList.add(draftOrderElement);
+                }
+            }
+        );
+        caseDataMap.put("rejectedOrdersDynamicList", ElementUtils.asDynamicList(
+            supportedDraftOrderList,
+            null,
+            DraftOrder::getLabelForOrdersDynamicList
+        ));
+        log.info("*** rejected orders dynamic list : {}", caseDataMap.get("rejectedOrdersDynamicList"));
+        log.info("*** draftoo order collection : {}", caseData.getDraftOrderCollection());
+        return caseDataMap;
+    }
+
+    public Map<String, Object> populateInstructionsAndDocuments(CaseData caseData) {
+        Map<String, Object> caseDataMap = new HashMap<>();
+        DraftOrder selectedOrder = getSelectedDraftOrderDetails(caseData.getDraftOrderCollection(), caseData.getManageOrders()
+            .getRejectedOrdersDynamicList());
+        caseDataMap.put(ORDER_NAME, ManageOrdersUtils.getOrderName(selectedOrder));
+        caseDataMap.put("orderUploadedAsDraftFlag", selectedOrder.getIsOrderUploadedByJudgeOrAdmin());
+        caseDataMap.put("orderType", selectedOrder.getOrderType());
+        caseDataMap.put(CASE_TYPE_OF_APPLICATION, caseData.getCaseTypeOfApplication());
+        caseDataMap.put("manageOrderOptionType", selectedOrder.getOrderSelectionType());
+        caseDataMap.put(IS_ORDER_CREATED_BY_SOLICITOR, selectedOrder.getIsOrderCreatedBySolicitor());
+        if (selectedOrder.getOtherDetails().getInstructionsToLegalRepresentative() != null) {
+            caseDataMap.put("instructionsToLegalRepresentative", selectedOrder.getOtherDetails().getInstructionsToLegalRepresentative());
+        }
+        caseDataMap.put(
+            IS_HEARING_PAGE_NEEDED,
+            isHearingPageNeeded(selectedOrder.getOrderType(), selectedOrder.getC21OrderOptions()) ? Yes : No
+        );
+        caseDataMap.put(CASE_TYPE_OF_APPLICATION, CaseUtils.getCaseTypeOfApplication(caseData));
+        if (YesOrNo.Yes.equals(selectedOrder.getIsOrderUploadedByJudgeOrAdmin())) {
+            caseDataMap.put("editOrderTextInstructions", "Open the draft order and edit it to include the judge's instructions");
+            caseDataMap.put("previewUploadedOrder", selectedOrder.getOrderDocument());
```

Comment:
What about welsh document?

Reply:
This is uploaded order 
Not the generated docs

Reply:
They are not required to be displayed to solicitor as per the design